src/@ionic-native/plugins/zeroconf/index.ts

```diff
@@ -0,0 +1,132 @@
+import { Injectable } from '@angular/core';
+import { Cordova, Plugin } from '@ionic-native/core';
+import { Observable } from 'rxjs/Observable';
+
+export interface ZeroconfService {
+  domain: string;
+  type: string;
+  name: string;
+  port: number;
+  hostname: string;
+  ipv4Addresses: Array<string>;
+  ipv6Addresses: Array<string>;
+  txtRecord: any;
+}
+
+export interface ZeroconfResult {
+  action: 'registered' | 'added' | 'removed';
+  service: ZeroconfService;
+}
+
+/**
+ * @name Zeroconf
+ * @description
+ * This plugin allows you to browse and publish Zeroconf/Bonjour/mDNS services.
+ * @usage
+ * ```typescript
+ * import { Zeroconf } from '@ionic-native/zeroconf';
+ *
+ * constructor(private zeroconf: Zeroconf) { }
+ *
+ * ...
+ * ```
+ */
+@Plugin({
+  pluginName: 'Zeroconf',
+  plugin: 'cordova-plugin-zeroconf',
+  pluginRef: 'cordova.plugins.zeroconf',
+  repo: 'https://github.com/becvert/cordova-plugin-zeroconf'
+})
+@Injectable()
+export class Zeroconf {
+  /**
+   * Returns this device's hostname.
+   * @return {Promise<string>}
+   */
+  @Cordova()
+  getHostname(): Promise<string> { return; }
+
+  /**
+   * Publishes a new service.
+   * @usage
```

Comment:
Move all the `@usage` documentation to the main `@usage` block in the class documentation. 

The usage here will not be rendered in the docs page.

Please apply this fix to all methods below. Simply cut & paste them above.